backend/backend/asgi.py

```diff
@@ -8,13 +8,17 @@
 
 import os
 
-from django.core.asgi import get_asgi_application
-from dotenv import load_dotenv
+from dotenv import find_dotenv, load_dotenv
+
+load_dotenv(find_dotenv() or "")
```

Comment:
@hari-kuriakose isn't this the default params / behaviour of the `load_dotenv()` ? 
1. Why do we call it this way?
1. Isi it necessary to call `load_dotenv()` again?